scipy/spatial/distance.py

```diff
@@ -1153,7 +1153,8 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
     Y : ndarray
         Returns a condensed distance matrix Y.  For
         each :math:`i` and :math:`j` (where :math:`i<j<n`), the
-        metric ``dist(u=X[i], v=X[j])`` is computed and stored in entry ``ij``.
+        metric ``dist(u=X[i], v=X[j])`` is computed and stored 
+        in the :math:`ij` th entry.
```

Comment:
@jaimefrio Thanks for pointing out, I actually forgot to change the important one :/. I think it should be changed in ``cdist`` too. We could use the same formula mentioned in the ``squareform`` docstring.